plugins/out_s3/s3.c

```diff
@@ -2104,6 +2129,10 @@ static void cb_s3_flush(struct flb_event_chunk *event_chunk,
     struct s3_file *upload_file = NULL;
     struct flb_s3 *ctx = out_context;
     struct multipart_upload *m_upload_file = NULL;
+    msgpack_unpacked result;
+    msgpack_object *obj;
+    size_t off = 0;
+    struct flb_time tms;
```

Comment:
Could you rename this to:
```
chunk_first_log_time
```

TMS could mean
- Current time
- Time of first log in file
- Time of first log in chunk

I think changing this to `chunk_first_log_time` would make our logic clearer and help us to avoid/discover logic problems

Reply:
ok